application/src/main/resources/ecc.yml

```diff
@@ -167,6 +167,16 @@ repair:
 
 statistics:
   enabled: true
+  ##
+  ## How metrics should be reported
```

Comment:
Some comment about what is the behavior if all are disabled but statistics is true would be good here. As well as if statistics is false and all are enabled.